src/javascript/app/pages/trade/charts/digit_info.js

```diff
@@ -95,21 +99,23 @@ const DigitInfo = (() => {
     };
 
     const addContent = (underlying) => {
-        const domain    = document.domain.split('.').slice(-2).join('.');
-        let underlyings = [];
-        const symbols   = Symbols.getAllSymbols();
-        Object.keys(symbols).forEach((key) => {
-            if (/^(R_|RD)/.test(key)) {
-                underlyings.push(key);
-            }
-        });
-        underlyings = underlyings.sort();
-        underlyings.splice(2, 0, '1HZ10V');  // add Volatility 10 (1s)
-        underlyings.splice(4, 0, '1HZ100V'); // add Volatility 100 (1s)
-        let elem    = '';
-        for (let i = 0; i < underlyings.length; i++) {
-            elem += `<option value="${underlyings[i]}">${symbols[underlyings[i]]}</option>`;
-        }
+        const domain  = document.domain.split('.').slice(-2).join('.');
+        const market  = Defaults.get('market');
+        const symbols = ActiveSymbols.getSymbols(State.getResponse('active_symbols'));
```

Comment:
maybe we can have a helper function for `getSymbolsForMarket`, currently we only have `getSymbolsForSubmarket`

Reply:
@mustofa-binary that's what I thought at first, but `getSymbolsForSubmarket` isn't usable from the outside, it's for populating symbols internally in that function. if you see the line `submarket.symbols = {};` in that function it makes it clear that it's for internal use. so we can keep this outside as it's just a filtering on symbols by market, not something very long or complicated.